docs/guide/content_rendering.md

```diff
@@ -1119,80 +1219,6 @@ Here again custom parameters can be used in the template, e.g.:
-
-``` yaml
-parameters:
-    my.custom.view_controller.class: Acme\TestBundle\MyViewController
-
-services:
-    my.custom.view_controller:
-        class: %my.custom.view_controller.class%
-        arguments: [@some_dependency, @other_dependency]
-
-    # Change the alias here and make it point to your own controller
-    ez_content:
-        alias: my.custom.view_controller
-```
-
-!!! caution
-
-    Doing so will completely override the built-in `ViewController`! Use this at your own risk!
-
-### Custom controller structure
-
-Your custom controller can be any kind of [controller supported by Symfony](http://symfony.com/doc/current/book/page_creation.html#step-2-create-the-controller) (including [controllers as a service](http://symfony.com/doc/current/cookbook/controller/service.html)).
-
-The only requirement here is that your action method must have a similar signature to `ViewController::viewLocation()` or `ViewController::viewContent()` (depending on what you're matching of course). However, note that not all arguments are mandatory, since [Symfony is clever enough to know what to inject into your action method](http://symfony.com/doc/current/book/routing.html#route-parameters-and-controller-arguments). That is why **you don't need to exactly imitate the `ViewController`'s signature**. For example, if you omit the `$layout` and `$params` arguments, it will still be valid. Symfony will just avoid injecting them into your action method.
-
-#### Built-in ViewController signatures
-
-``` php
-// viewLocation() signature
-/**
- * Main action for viewing content through a location in the repository.
- *
- * @param int $locationId
- * @param string $viewType
- * @param boolean $layout
- * @param array $params
- *
- * @throws \Symfony\Component\Security\Core\Exception\AccessDeniedException
- * @throws \Exception
- *
- * @return \Symfony\Component\HttpFoundation\Response
- */
-public function viewLocation( $locationId, $viewType, $layout = false, array $params = array() )
-```
-
-``` php
-// viewContent() signature
-/**
- * Main action for viewing content.
- *
- * @param int $contentId
- * @param string $viewType
- * @param boolean $layout
- * @param array $params
- *
- * @throws \Symfony\Component\Security\Core\Exception\AccessDeniedException
- * @throws \Exception
- *
- * @return \Symfony\Component\HttpFoundation\Response
- */
-public function viewContent( $contentId, $viewType, $layout = false, array $params = array() )
-```
-
-!!! caution "Caching"
-
-    When you use your own controller, **it is your responsibility to define cache rules**, like with every custom controller!
-
-    So don't forget to set cache rules and the appropriate `X-Location-Id` header in the returned `Response` object.
```

Comment:
This is still valid, for any kind of plain symfony controller.

Reply:
Ignore this for now, would only make sense to mention for symfomy controllers, and as that is not done here anymore it can be removed.

Todo as followup:
- update `docs/guide/http_cache.md` for 1.13 and up with latests updates in: https://github.com/ezsystems/ezplatform-http-cache/blob/master/docs/using_tags.md#how-response-tagging-is-done

Reply:
Is this "todo as a followup" a story we can plan for a next sprint?

Reply:
if doc team wants to take it, feel free, alternative was if @natanael89 wanted to update it.